app/models/ems_event.rb

```diff
@@ -189,8 +191,13 @@ def self.event_allowed_ems_ref_keys
   end
   private_class_method :event_allowed_ems_ref_keys
 
+  def self.event_blacklisted_keys
+    %w[ems_uid ems_type]
```

Comment:
This sounds like it's necessary because we are piggybacking on the event syndication, but I'm wondering if we should organize it differently, such that the collector puts an even without these keys in the first place, then a syndicator comes along and enhances those into a different queue.  In this way, we can also disable the event syndication entirely where it's not needed.  Thoughts?

Reply:
Yes that's correct this is solely necessary because of the syndication, and yeah I like that approach a lot.